.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
@lucasdemarchi I think the advantage of having it here is that it separates jobs for clang and gcc allowing to more easily see which one is failing.